nx.json

```diff
@@ -0,0 +1,18 @@
+{
+  "npmScope": "spartacus",
+  "affected": {
+    "defaultBase": "develop"
```

Comment:
Let's remove the default base config to keep a low overhead with the upcoming branching strategy where the default will change over time.